extensions/observability/common/src/main/java/io/quarkus/observability/common/config/VictoriaMetricsConfig.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.observability.common.config;
+
+import java.util.Optional;
+import java.util.Set;
+
+import io.quarkus.observability.common.ContainerConstants;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.smallrye.config.WithDefault;
+
+@ConfigGroup
+@DevTarget("io.quarkus.observability.devresource.victoriametrics.VictoriaMetricsResource")
+public interface VictoriaMetricsConfig extends ContainerConfig {
```

Comment:
Afais, Prometheus is up high there in the wish list, and VM == Prometheus.
I'm planning on adding real Prometheus asap.

And as such, I would leave PromQL -- it might come useful to some users.
Might probably need some more tests, but REST API was "copy/pasted" (and implemented) from the docs.

Reply:
I'm not contesting being here at some point, I'm just concerned about the size of the PR. We need to break this up because the important parts are totally obfuscated. 
I strongly believe we should have only Jaeger-all and Grafana-all on this first PR. 
Also believe that expanding the number of destinations is secondary and we should focus on making sure the experience is spot on.